dev-docs/publisher-api-reference.md

```diff
@@ -1290,4 +1280,77 @@ This call returns the following DFP video ad tag URL:
+| iu             | string | (Required) DFP adUnit ID.  For more information, see the [DFP documentation on `iu`](https://support.google.com/dfp_premium/answer/1068325?hl=en#iu)                                                                                                                                                               | `/19968336/prebid_cache_video_adunit`         |
+| cust_params    | object | (Optional) Key-value pairs that will be sent to DFP on the video ad tag URL.  Any key-values given here will be merged with Prebid.js' standard targeting key-values.  For more information, see the [DFP documentation on `cust_params`](https://support.google.com/dfp_premium/answer/1068325?hl=en#cust_params) | `{section: "blog", anotherKey: "anotherValue"}` |
+| "arbitraryKey" | string | (Optional) Any additional querystring parameters that will be used to construct the DFP video ad tag URL.                                                                                                                                                                                                          | `output: "vast"`                              |
+| `description_url` | string | (Optional) Describes the video.  Required for Ad Exchange.  Prebid.js will build this for you unless you pass it explicitly.
+
+{: .alert.alert-info :}
+Note: Prebid.js will choose reasonable default values for any required DFP URL parameters that are not included in the `options.params` object.
+
+For more information about the options supported by the DFP API, see [the DFP API docs](https://support.google.com/dfp_premium/answer/1068325?hl=en#env).
+
+<a name="buildVideoUrl-usage-scenarios-and-expected-behavior" />
+
+#### Usage scenarios and expected behavior
+
+Your (optional) usage of Prebid Cache, a server-side asset cache hosted by Prebid.org, may change how the method is used, and how bidders are required to respond, depending on the argument type ([`options.params`](#options-params-argument), [`options.url`](#options-url-argument)), or both.  For more information, see [Example Usage](#buildVideoUrl-example-usage) and [Usage scenarios and expected behavior](#buildVideoUrl-usage-scenarios-and-expected-behavior).
+
+Reasons you might pass the `options.url` field include the following use cases:  
+- You are not using an ad server, and you want to insert the winning VAST content into your player directly  
+- You are using a video ad server that can accept a standalone VAST URL, as opposed to having to pass the VAST URL as a query string parameter  
+
+
+The table below lists the expected behavior of this method when called with various configurations.
+
+<span class="FIXME" style="color: rgb(255,0,0);">FIXME</span>: In the PR comment (https://github.com/prebid/Prebid.js/pull/1663) options.params with cache DISABLED is not mentioned.  Does it work?
```

Comment:
Yes, if the correct adserver adunit id is used. In our example case, `/19968336/prebid_cache_video_adunit` is the adunit id used when prebid-cache is enabled, which is correct. If prebid-cache is disabled, that same adunit id will no longer work, but `/19968336/prebid_video_adunit` will because it is expecting a vast url rather than a prebid-cache key